porting/nimble/src/os_mempool.c

```diff
@@ -38,7 +38,8 @@
 #define OS_MEMPOOL_TRUE_BLOCK_SIZE(mp) OS_MEM_TRUE_BLOCK_SIZE(mp->mp_block_size)
 #endif
 
-STAILQ_HEAD(, os_mempool) g_os_mempool_list;
+STAILQ_HEAD(, os_mempool) g_os_mempool_list =
+    STAILQ_HEAD_INITIALIZER(g_os_mempool_list);
```

Comment:
Could you explain why is this needed?  (AFAIR initially it was like that to allow multiple initialization while executing tests)